torch/_torch_docs.py

```diff
@@ -2698,6 +2707,8 @@
 
 If :attr:`mat` is a `n x m` Tensor, :attr:`vec` is a 1D Tensor of size `m`, :attr:`out` will be 1D of size `n`.
 
+.. note:: This function does not :ref:`broadcast <broadcasting-semantics>`.
```

Comment:
Do we want our users to migrate to use `matmul` instead of the individual `mm`, `bmm`, `ger`, etc? Are there benefits on using these low-level functions compared to `matmul`?
If not, then I'd say to mention in the docs that these functions are internal or low-level, and point to the `matmul` documentation as well (not only for `mm` and `bmm`, but also `ger` ad `mv`.
What do you think?